packages/metrics/tests/unit/coldStartMetric.test.ts

```diff
@@ -65,27 +65,62 @@ describe('ColdStart metric', () => {
     );
   });
 
-  it('includes the function name in the cold start metric', () => {
+  it('does not override the function name from constructor in the cold start metric', () => {
```

Comment:
In general, I don't like to change existing tests with new behavior, but rather preserve the existing one and add a new one with new behavior. I read the tests and I don't know if you have a test explicitly covering this.

Reply:
The diff is a bit confusing because we renamed the test description and added more tests.

The difference in the test is that before we were using the now marked as deprecated method, while now we use the new one.

Either way point taken.

Reply:
Thanks for explaining.